spokestack/io/pyaudio.py

```diff
@@ -3,6 +3,10 @@
 """
 import numpy as np  # type: ignore
 import pyaudio  # type: ignore
+from streamp3 import MP3Decoder  # type: ignore
```

Comment:
we don't want mp3 or tts details in the pyaudio stuff
this module should just be a simple abstraction around pyaudio